Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    terrafying (1.1.4)
+    terrafying (0.0.0)
```

Comment:
this is definitely terrafying 2.0.0

Reply:
That happened because of https://github.com/uswitch/terrafying/pull/80/files#diff-f72f7c0ad6a78bbf1b8eb71724736f0bR2

I'd be curious of some opinion on whether we could be doing CI pushing to RubyGems on tag better?

Reply:
We could be doing that yes; as a gem, we don't really care about our gemfile.lock though so that's okay. I'd like to have it in rubygems properly for sure :)